test_sets/Defects/Defects_release_7_2.txt

```diff
@@ -2,3 +2,4 @@
 ./test_scripts/Defects/7_2/3556_2_GetListOfPermission_contains_groups_with_user_consent_prompt.lua
 ./test_scripts/Defects/7_2/977_SDL_sends_appName_in_vrSynonyms_and_ttsName_in_case_of_lower_and_upper_bound_values_of_params_in_json_file.lua
 ./test_scripts/Defects/7_2/2398_SendLocation_with_one_OR_more_allowed_params_and_one_OR_more_disallowed_params_by_user.lua
+./test_scripts/Defects/7_2/2380_PoliciesManager_must_allow_all_requested_VD_params_in_case_parameters_field_is_omitted.lua
```

Comment:
Propose to make this new script to be part of VD test set (`./test_sets/API/vehicle_data.txt`) and move it to `test_scripts/API/VehicleData/OnVehicleData` folder